### ISSUE TITLE
weird white headers in side bar since 0.10.5

### ISSUE THREAD

**Issue by bitbonk** [psu/logseq-style-tweaks#7]
After I updated to 0.10.5 (don't know which version I came from) I have these weird white headers in the side bar (light mode):

![image](https://github.com/psu/logseq-style-tweaks/assets/418030/e359bcb6-ed86-45b4-9dca-896fbfda57f5)

In dark mode it also looks a bit weird, IMHO:

![image](https://github.com/psu/logseq-style-tweaks/assets/418030/5e423a95-c634-463d-9d0f-9fb50ba1a072)

**Comment by psu** [comment]
Yeah you are probably right. I’m not maintaining this anymore. /PontusOn 22 Jan 2024, at 11:22, bitbonk ***@***.***> wrote:﻿
After I updated to 0.10.5 (don't know which version I came from) I have these weird white headers in the side bar (light mode):
image.png (view on web)
In dark mode it also looks a bit weird, IMHO:
image.png (view on web)
